src/client/src/services/credit/mockNewRfqs.ts

```diff
@@ -0,0 +1,105 @@
+import { concat, delay, from } from "rxjs"
+import {
+  PASSED_QUOTE_STATE,
+  QUOTE_PASSED_RFQ_UPDATE,
+  RfqUpdate,
+  Direction,
+  RfqState,
+  QUOTE_UPDATED_RFQ_UPDATE,
+  RFQ_CREATED_RFQ_UPDATE,
+  PENDING_WITH_PRICE_QUOTE_STATE,
+  PENDING_WITHOUT_PRICE_QUOTE_STATE,
+} from "@/generated/NewTradingGateway"
+
+const dealerIdLength = 10
+const AmountOfRFQs = 10
+const mockRFQs: RfqUpdate[] = []
+
+const StartState = () => {
+  return from(constructMockStartStateRFQS())
+}
+const constructMockStartStateRFQS = () => {
```

Comment:
Having this global array `mockRFQs` feels pretty strange to me, maybe you could have each of the construction functions return an array and spread them all together to eliminate that floating global.

```
return [
  ...constructRfqCreatedRfqUpdate(),
  ...constructRfqUpdate(),
  ...addStateOfTheWorld()
]
```